pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/EasyPostUtil.java

```diff
@@ -206,7 +208,25 @@ public Address createAddress(DDPParticipant ddpParticipant, @NonNull String phon
         toAddressMap.put(this.country, ddpParticipant.getCountry());
         toAddressMap.put(this.phone, phone); //Needed for FedEx!
         toAddressMap.put(this.residential, true);
+        return toAddressMap;
+    }
 
+    public Address createAddress(DDPParticipant ddpParticipant, @NonNull String phone, @NonNull String company)
+            throws EasyPostException {
+        if (StringUtils.isEmpty(this.phone)) {
+            throw new RuntimeException("Contact phone number is needed");
+        }
+        Map<String, Object> toAddressMap = createAddressMap(ddpParticipant, phone);
+        toAddressMap.put(this.company, company); // Care Of field goes here
+        return Address.create(toAddressMap);
+    }
+
+    public Address createAddress(DDPParticipant ddpParticipant, @NonNull String phone)
+            throws EasyPostException {
+        if (StringUtils.isEmpty(this.phone)) {
+            throw new RuntimeException("Contact phone number is needed");
+        }
+        Map<String, Object> toAddressMap = createAddressMap(ddpParticipant, phone);
         return Address.create(toAddressMap);
```

Comment:
We can only call Address.create once, otherwise the company field won't be added if we add it later. So I created the `createAddressMap` to make the hashmap, and then we can add the company field if there is one